components/edit-collective/sections/Policies.js

```diff
@@ -546,6 +558,46 @@ const Policies = ({ collective }) => {
             />
           </P>
 
+          <StyledInputField
+            name="policies.EXPENSE_POLICIES.grantPolicy"
+            htmlFor="policies.EXPENSE_POLICIES.grantPolicy"
+            error={formik.errors.policies?.EXPENSE_POLICIES?.grantPolicy}
+            disabled={isSubmittingSettings}
+            labelProps={{ mb: 2, pt: 2, lineHeight: '18px', fontWeight: 'bold' }}
+            label={formatMessage(messages['grantExpensePolicy.label'])}
+          >
+            {inputProps =>
+              loading ? (
+                <LoadingPlaceholder height={50} width={1} />
+              ) : (
+                <RichTextEditor
+                  key={data?.account?.policies?.EXPENSE_POLICIES?.grantPolicy}
+                  data-cy="grant-expense-policy-input"
+                  withBorders
+                  showCount
+                  maxLength={EXPENSE_POLICY_MAX_LENGTH}
+                  error={formik.errors.policies?.EXPENSE_POLICIES?.grantPolicy}
+                  version="simplified"
+                  editorMinHeight="12.5rem"
+                  editorMaxHeight={500}
+                  id={inputProps.id}
+                  inputName={inputProps.name}
+                  onChange={formik.handleChange}
+                  placeholder={formatMessage(messages['grantExpensePolicy.placeholder'])}
+                  defaultValue={data?.account?.policies?.EXPENSE_POLICIES?.grantPolicy || ''}
+                  fontSize="14px"
+                  maxHeight={600}
+                />
+              )
+            }
+          </StyledInputField>
+          <P fontSize="14px" lineHeight="18px" color="black.600" my={2}>
+            <FormattedMessage
+              id="collective.expensePolicy.description"
+              defaultMessage="It can be daunting to file an expense if you're not sure what's allowed. Provide a clear policy to guide expense submitters."
```

Comment:
```suggestion
              id="collective.grantExpensePolicy.description"
              defaultMessage="It can be daunting to request a grant if you're not sure what's allowed. Provide a clear policy to guide grant requesters."
```